source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -234,8 +234,8 @@ namespace PlayFab
         batchCounter++;
 
         this->batch.clear(); // batch vector will be reused
-        this->batch.reserve(this->settings->maximalNumberOfItemsInBatch);
-        if(this->settings->emitType == PlayFabEventPipelineType::PlayFabPlayStream)
+        this->batch.reserve(this->m_settings->maximalNumberOfItemsInBatch);
```

Comment:
Please add a line before your conditional block begins - it makes the code easier to read

Reply:
Due to other comments, I've reverted all changes in this file.
So, I'm going to won't-fix this one, while acknowledging that you're totally right, but it's not right for this PR.